### ISSUE TITLE
 Can't run because no spec files were found.

### ISSUE THREAD

**Issue by granite** [mbolotov/intellij-cypress#62]
I just installed Cypress Pro plugin (Version 2.4.0) in Webstorm (Version 2021.2.1).  I tried to follow this video to setup a Test Run Configuration (ref: https://www.youtube.com/embed/1gjjy0RQeBw ) , but it seems out-of-date.  I don't see any 'Create Run Configuration' option in the context menu.  Instead I have More Run/Debug->Edit Run Configuration, but that didn't work.  So then I then went to the main menu Run->Configurations and clicked + -> Cypress.

I setup the working directory to be where all my cypress config and tests are located;  The directory structure is like this:
```
cypress
-> config
-> fixtures
-> integration
-> plugins
-> support
```

My tests are under integration.  When I try to run my test for one the spec files (`cypress\integration\user-profile\user-profile\user-profile.spec.js`) , I get an error saying it can't find the spec file,, even though all the paths look correct

```
"C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npx-cli.js" cypress run -b chrome --config-file config/cypress.json --reporter C:\Users\username\AppData\Local\Temp\intellij-cypress-reporter1099357078825348125.js --spec C:\Users\username\git_username\dwp-pwa\cypress\integration\user-profile\user-profile\user-profile.spec.js

Can't run because no spec files were found.

We searched for any files matching this glob pattern:

integration\user-profile\user-profile\user-profile.spec.js

Relative to the project root folder:

C:\Users\username\git_username\dwp-pwa\cypress

Process finished with exit code 1
```

![image](https://user-images.githubusercontent.com/2415272/130925202-38a4db8d-3658-4475-831a-f3008115a66c.png)

![image](https://user-images.githubusercontent.com/2415272/130925710-44364e2e-7785-49a6-898f-d3b1ddc094fa.png)

**Comment by mbolotov** [comment]
Hi @granite!
You can try the 2.5.0 version of plugin and define your specific path to the cypress configuration file:
![image](https://user-images.githubusercontent.com/19785715/135406088-34e4ca94-90a6-4841-b7cf-a70cb93261af.png)

It should fix the issue #1 from above

**Comment by mbolotov** [comment]
Hi @granite!
Have you had a chance to try the 2.5.0 version and specify a custom cypress config file path?
Does it solve your problem?
